File: Logika & Algoritma Pemrograman/Pertemuan6/src/Tugas.pas
program Hitung_Gaji_Mingguan;
uses crt;
var 
  nama : string;
  jam : integer;
  total : longint;

begin
  clrscr;
  write('Nama Karyawan      : '); readln(nama);
  write('Jumlah Jam kerja   : '); readln(jam);
  
  if jam < 48 then
    total := 2000 * jam
  else
    total := (48 * 2000) + ((jam - 48) * 3000);

  writeln('Jumlah Gaji dari karyawan ', nama, ' adalah ', total);
  readln;
end.

File: Semester 1/Logika & Algoritma Pemrograman/Pertemuan6/src/tugas.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
  string name;
  int jam, total;

  cout << "Masukan Nama Anda : "; cin >> name;
  cout << "Masukan Jam kerja Anda : "; cin >> jam;

  if (jam < 48) {
    total = jam * 2000;
  }else if (jam >= 48 ) {
    total = (48 * 2000) + ((jam - 48) * 3000);
  }

  cout << "Jumlah Gaji dari Karyawan " << name << " adalah " << total << endl;

  return 0;
}
